code/modules/fishing/fishing_gear.dm

```diff
@@ -350,3 +350,341 @@ TYPEINFO(/obj/item/fish_portal)
+				src.UpdateIcon()
+				user.update_inhands()
+				src.lure.pixel_x = rand(-12, 12)
+				src.lure.pixel_y = rand(-12, 12)
+				src.lure.set_loc(get_turf(src.loc))
+				src.lure.throw_at(target, src.line_length, 2)
+			else
+				src.is_fishing = FALSE
+				src.UpdateIcon()
+				user.update_inhands()
+				src.lure.throw_at(src, 15, 2)
+				SPAWN(0.2 SECONDS)
+					if (src.lure)
+						if (src.lure.owner)
+							src.lure.owner.throw_at(src, 2, 2)
+						else
+							src.lure.set_loc(src)
+
+	update_icon()
+		//state for fishing
+		if (src.is_fishing)
+			src.icon_state = "syndie_fishing_rod-active"
+			src.item_state = "syndie_fishing_rod-active"
+		//state for not fishing
+		else
+			src.icon_state = "syndie_fishing_rod-inactive"
+			src.item_state = "syndie_fishing_rod-inactive"
+
+	disposing()
+		. = ..()
+		qdel(src.lure)
+
+	// reels in, returns whether damage was dealt
+	proc/reel_in(mob/target, mob/user, damage_on_reel = 7)
+		target.setStatusMin("staggered", 4 SECONDS)
+		if(BOUNDS_DIST(target, user) == 0)
+			if (issilicon(target))
+				user.visible_message("<span class='alert'><b>[user] tears some scrap out of [target] with \the [src.name]!</b></span>")
+				playsound(target.loc, 'sound/impact_sounds/circsaw.ogg', 40, 1)
+				random_burn_damage(target, damage_on_reel)
+			else
+				user.visible_message("<span class='alert'><b>[user] reels some meat out of [target] with \the [src.name]!</b></span>")
+				playsound(target.loc, 'sound/impact_sounds/Flesh_Tear_2.ogg', 50, 1)
+				take_bleeding_damage(target, user, damage_on_reel, DAMAGE_CUT)
+			random_brute_damage(target, damage_on_reel)
+			if (damage_on_reel >= 25)
+				target.changeStatus("weakened", sqrt(damage_on_reel) / 3 SECONDS)
+				target.force_laydown_standup()
+				if (target.bioHolder && target.bioHolder.Uid && target.bioHolder.bloodType)
+					gibs(target.loc, blood_DNA=target.bioHolder.Uid, blood_type=target.bioHolder.bloodType, headbits=FALSE, source=target)
+				else
+					gibs(target.loc, headbits=FALSE, source=target)
+			return TRUE
+		else
+			step_towards(target, user)
+			return FALSE
+
+/obj/item/implant/syndie_lure
+	name = "minnow"
+	desc = "One of the most common bait fish, looks like this one got away! Until it caught you."
+	icon = 'icons/obj/foodNdrink/food_fish.dmi'
+	icon_state = "minnow"
+	object_flags = NO_GHOSTCRITTER
+	throwforce = 5
+	density = 0
+	var/obj/item/syndie_fishing_rod/rod = null
+
+	UpdateName()
+		src.name = "[name_prefix(null, 1)][src.real_name][name_suffix(null, 1)]"
+
+	attackby(obj/item/W, mob/user)
+		if (!(W == src.rod) && isliving(user) && !user.nodamage && GET_DIST(src, src.rod) < src.rod.line_length)
+			user.changeStatus("weakened", 5 SECONDS)
+			user.TakeDamage(user.hand == LEFT_HAND ? "l_arm": "r_arm", 15, 0, 0, DAMAGE_STAB)
+			user.force_laydown_standup()
+			implanted(user, null)
+		else
+			return ..()
+
+	attack_hand(mob/user)
+		if (isliving(user) && !user.nodamage && GET_DIST(src, src.rod) < src.rod.line_length)
```

Comment:
Kind of weird that you can safely pick up the lure when it's out of range. Wouldn't it make more sense if the lure was returned when the rod moved too far away from it? See outermost movable signals.

Reply:
Thank you, I wasn't sure how to handle that.